typescript/formsApps.d.ts

```diff
@@ -452,6 +453,7 @@ export type FormsAppConfiguration<
   samlIdentityProviderName?: string | null
   logoutRedirectUrl?: string
   styles: T
+  taskGroupInstances?: ScheduledTasksTypes.TaskGroupInstance[]
```

Comment:
Shouldn't this be where all the other task config is for an app? Or are we also moving/have moved where tasks are stored on an app? Memory serves its in `styles` right now?

Reply:
Need the label for each instance for navigation purposes, styles wont have that anymore.